helm/botkube/values.yaml

```diff
@@ -51,8 +51,8 @@ kubeconfig:
   existingSecret: ""
 
 
-# -- Map of enabled sources. The `sources` property name is an alias for a given configuration.
-# Key name used as a binding reference.
+# -- Map of enabled sources. The property name under `sources` object is an alias for a given configuration. You can define multiple sources configuration with different names.
+# Key name is used as a binding reference.
```

Comment:
maybe it would be good to add also a default description. Currently, we print the whole object in README.md which is not readable.

We can define sth like
```
# @default -- Enables watching for Kubernetes events and sending recommendations.
```
The property name is "clickable" and will navigate to the definition, which will provide more context.

Reply:
@mszostok That would be a duplication of the description, but I like the idea to get rid of the full object. WDYT about ```
# @default -- See the `values.yaml` file for full object.
```
?

Reply:
>That would be a duplication of the description

Not exactly, as there we're saying what kind of k8s notification we enable. However `# @default -- See the `values.yaml` file for full object.` also works for me 👌

Reply:
Alright then, I will append that to the actual description to make sure we have this info in docs 👍